Formula/wownero.rb

```diff
@@ -45,22 +47,18 @@ class Wownero < Formula
 
   conflicts_with "monero", because: "both install a wallet2_api.h header"
 
-  # patch build issue (missing includes)
-  # remove when wownero syncs fixes from monero
-  patch do
-    url "https://github.com/monero-project/monero/commit/96677fffcd436c5c108718b85419c5dbf5da9df2.patch?full_index=1"
-    sha256 "e39914d425b974bcd548a3aeefae954ab2f39d832927ffb97a1fbd7ea03316e0"
-  end
-
   def install
+    args = std_cmake_args
```

Comment:
Ideally we should prepend rather than append to `std_cmake_args` (or `std_meson_args`) to make sure we're never overriding flags in it. Or at least be clearer about when we're doing that by only appending whenever we intend to override.